app/src/main/res/layout/activity_report_profile.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Nice job with this minimalist report UI as it makes it more intuitive.
However, maybe adding report categories could make the misbehavior clearer ?

Reply:
I agree, as this approach could help with giving unique attention for each problem recorded

Reply:
Yes, we can do that for the next sprint